src/patternfly/components/Table/examples/table-expandable-example.hbs

```diff
@@ -47,19 +47,22 @@
     {{/table-tr}}
 
     {{#> table-tr table-tr--expandable="true" table-tr--IsExpanded="true"}}
-      {{#> table-td table-td--attribute=(concat 'colspan="7" id="' table--id '-content1"')}}
+      {{#> table-td}}{{/table-td}}
+      {{#> table-td}}{{/table-td}}
+      {{#> table-td table-td--attribute=(concat 'colspan="4 id="' table--id '-content1"')}}
         <div class="pf-c-table__expandable-row-content">
```

Comment:
should we create an hbs file for this? we use it a lot.

Reply:
yes created one